src/store/reducers/members.ts

```diff
@@ -251,6 +312,62 @@ function applyOperation(
       return prevState;
     }
     switch (operation.op_code) {
+      case OperationType.CREATE_MEMBER: {
+        const {
+          full_name,
+          username,
+          request_invite_from_member_id
+        } = operation.data;
+        const memberData = {
+          memberId: creator_uid,
+          username: username,
+          fullName: full_name,
+          createdAt: new Date(created_at),
+          inviteConfirmed: false,
+          lastMinted: new Date(created_at)
+        };
+
+        // TODO handle request_invite_from_member_id property
+        // assertMemberIdPresentInState(prevState, request_invite_from_member_id, operation);
+
+        assertMemberIdNotPresentInState(prevState, creator_uid, operation);
+
+        const newMember = new Member({
+          ...memberData
+        });
+
+        return addMemberToState(prevState, newMember);
+      }
+      case OperationType.REQUEST_VERIFICATION: {
+        const { to_uid } = operation.data;
+
+        assertMemberIdPresentInState(prevState, creator_uid, operation);
```

Comment:
Would it make sense to move all these `assertMemberIdPresentInState` into operationIsRelevantAndValid? Just at the beginning have something like if op in SET_CONST do the two asxertions?

Reply:
So long as the ids are predictable that would be fine

Reply:
I'll leave a TODO in for this for now - don't feel like modifying all the ops for this PR.